### ISSUE TITLE
[Bug]: DAV is slow

### ISSUE THREAD

**Issue by MrGumpIT** [LinwoodDev/Butterfly#685]
### What happened?

when I use the DAV connection everything is so slow, the UI lags a lot, I think you have to change approach, work on local files and when they change synchronize them

### Version

2.1.0-rc.1

### Platform

Android

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by CodeDoctorDE** [comment]
yeah it's not ideal. Feel free to improve it (it's inside `file_system_dav.dart`)

**Comment by CodeDoctorDE** [comment]
Hello,
is this issue present in the current version? There were a few refactors during that time

**Comment by CodeDoctorDE** [comment]
should be fixed in https://github.com/LinwoodDev/Butterfly/issues/927
